superset-frontend/src/SqlLab/components/SqlEditor/index.tsx

```diff
@@ -759,6 +760,10 @@ const SqlEditor: React.FC<Props> = ({
     );
   };
 
+  const handleCursorPositionChange = (newPosition: any) => {
```

Comment:
Since you already added[ type for the cursor position,](https://github.com/apache/superset/pull/26754/files#diff-98bfde8ed111b1aa1a94f5da25ed87585b19908a628063fd133958e9576c239aR38) please use `CursorPosition` instead of any type

```suggestion
  const handleCursorPositionChange = (newPosition: CursorPosition) => {
```